package.json

```diff
@@ -1,60 +1,66 @@
-    "url": "git+ssh://git@github.com/httptoolkit/brotli-wasm.git"
-  },
-  "keywords": [
-    "brotli",
-    "wasm",
-    "compression",
-    "decompression"
-  ],
-  "author": "Tim Perry <tim@httptoolkit.tech>",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/httptoolkit/brotli-wasm/issues"
-  },
-  "homepage": "https://github.com/httptoolkit/brotli-wasm#readme",
-  "devDependencies": {
-    "@types/chai": "^4.2.18",
-    "@types/mocha": "^8.2.2",
-    "@types/node": "^15.6.0",
-    "buffer": "^6.0.3",
-    "chai": "^4.3.4",
-    "karma": "^6.3.2",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-mocha": "^2.0.1",
-    "karma-sourcemap-loader": "^0.3.8",
-    "karma-spec-reporter": "0.0.32",
-    "karma-typescript": "^5.5.1",
-    "karma-webpack": "^5.0.0",
-    "mocha": "^8.4.0",
-    "shelljs": "^0.8.4",
-    "ts-loader": "^9.2.1",
-    "ts-node": "^9.1.1",
-    "typescript": "^4.2.4",
-    "wasm-pack": "^0.9.1",
-    "webpack": "^5.37.1"
-  }
+    "name": "brotli-wasm",
+    "version": "1.2.0",
+    "description": "A reliable compressor and decompressor for Brotli, supporting node & browsers via wasm",
+    "main": "./pkg.node/brotli_wasm.js",
+    "types": "./pkg.node/brotli_wasm.d.ts",
+    "browser": "./index.browser.js",
+    "sideEffects": false,
+    "files": [
+        "pkg.node",
+        "pkg.web",
+        "pkg.bundler",
+        "index.browser.js"
+    ],
+    "scripts": {
+        "build": "node ./build.js",
+        "test": "npm run test:node && npm run test:browser",
+        "test:node": "TS_NODE_FILES=true mocha -r ts-node/register/transpile-only 'test/**/*.spec.ts'",
+        "test:browser": "karma start",
+        "test:browser:debug": "npm run test:browser -- --single-run=false --browsers Chrome"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+ssh://git@github.com/httptoolkit/brotli-wasm.git"
+    },
+    "keywords": [
+        "brotli",
+        "wasm",
+        "compression",
+        "decompression"
+    ],
+    "author": "Tim Perry <tim@httptoolkit.tech>",
+    "license": "Apache-2.0",
+    "bugs": {
+        "url": "https://github.com/httptoolkit/brotli-wasm/issues"
+    },
+    "homepage": "https://github.com/httptoolkit/brotli-wasm#readme",
+    "devDependencies": {
+        "@types/chai": "^4.3.1",
+        "@types/mocha": "^9.1.1",
+        "@types/node": "^18.6.3",
+        "buffer": "^6.0.3",
+        "chai": "^4.3.6",
+        "karma": "^6.4.0",
+        "karma-chai": "^0.1.0",
+        "karma-chrome-launcher": "^3.1.1",
+        "karma-mocha": "^2.0.1",
+        "karma-sourcemap-loader": "^0.3.8",
+        "karma-spec-reporter": "0.0.34",
+        "karma-typescript": "^5.5.3",
+        "karma-webpack": "^5.0.0",
+        "mocha": "^10.0.0",
+        "shelljs": "^0.8.5",
+        "ts-loader": "^9.3.1",
+        "ts-node": "^10.9.1",
+        "typescript": "^4.7.4",
+        "webpack": "^5.74.0"
+    },
+    "prettier": {
```

Comment:
I tried to match the existing style that you chose for this project but there were some places where mixed styles were used so I just picked the one that was used most frequently